.github/main.workflow

```diff
@@ -61,6 +61,12 @@ action "Push image to GCR" {
   args = ["docker push gcr.io/$PROJECT_ID/$APPLICATION_NAME"]
 }
 
+action "Delete old images from GCR" {
+  needs = ["Push image to GCR"]
```

Comment:
We should move this to the end of the pipeline, before we merge it, shouldn't we?